### DISCUSSION TITLE
cell voltages / Current

### DISCUSSION THREAD

**Discussion by rolandlueders** [Louisvdw/dbus-serialbattery#986] [Ideas]
I have tree 24V/300Ah batteries with JK-BMSes to a Rasberry4. There ist installed "serialbattery" and "Battery Aggregator". It would be very nice, when the battery current is displayd in the "Cells Volteges" page in the fist line. Like "Cells Sum      26.43V  @12.31A"
Roland

**Comment by mr-manuel** [comment]
We will not do that.
